catalyst-toolbox/src/bin/cli/snapshot/mod.rs

```diff
@@ -22,9 +25,19 @@ pub struct SnapshotCmd {
     #[structopt(short, long)]
     threshold: Value,
 
-    /// Discrimination to use for initial addresses
+    /// Voter group to assign direct voters to.
+    /// If empty, defaults to "voter"
+    #[structopt(short, long)]
+    direct_voters_group: Option<String>,
+
+    /// Voter group to assign representatives to.
+    /// If empty, defaults to "rep"
     #[structopt(short, long)]
-    discrimination: Discrimination,
+    representatives_group: Option<String>,
+
+    /// Url of the representative db api server
+    #[structopt(short, long)]
+    reps_db_api_url: String,
```

Comment:
what about using `Url` from the `url` crate?

Reply:
probably better 👍🏼